src/entries/popup/handlers/wallet.ts

```diff
@@ -342,14 +343,15 @@ export const importAccountAtIndex = async (
     case 'Trezor':
       {
         const path = getHDPathForVendorAndType(index, 'Trezor');
-        const result = await window.TrezorConnect.ethereumGetAddress({
+        // @ts-expect-error --- the type definitons make no sense for this method.
+        // 'address' is required but it should be optional and only used for validation
+        const result = await TrezorConnect.ethereumGetAddress({
```

Comment:
Usage in their docs suggests this type is wrong. https://docs.trezor.io/trezor-suite/packages/connect/methods/ethereumGetAddress.html

Reply:
And it works ofc